lib/resources/example_json/soarian_examples_care_plan.rb

```diff
@@ -0,0 +1,178 @@
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "papi029"
+                    },
+                    {
+                        "url": "Message",
+                        "valueString": "Information provided is from 2017-02-09 through 2017-03-10 inclusive."
+                    }
+                ]
+            }
+        },
+        {
+            "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+            "valueExtension": {
+                "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "papi026"
+                    },
+                    {
+                        "url": "Message",
+                        "valueString": "The information provided is the latest configured data available at the time of charting and may not be reflective of the current clinical state of the care plan. It is the responsibility of the patient or patient representative to confirm care plan status with appropriate care providers."
+                    }
+                ]
+            }
+        }
+    ],
+    "type": "searchset",
+    "link": [
+        {
+            "relation": "self",
+            "url": "base-url/123abc/CarePlan?patientId=D4292B7B-AEDF-4CEF-B783-BD3AA1B0DD27&_format=json"
+        }
+    ],
+    "entry": [
+        {
+            "resource": {
+                "resourceType": "CarePlan",
+                "text": {
+                    "status": "additional",
+                    "div": "&lt;ul ccdRefId=\'Zcombined_NRS03.Assessments.95442.70425\'>&lt;li>MU3 Assessment123 : &lt;HTML>&lt;HEAD>&lt;style type=\'text/css\'>BODY,TD,TH,BUTTON,INPUT,SELECT,TEXTAREA{FONT-SIZE: 10pt; FONT-FAMILY: Arial,Helvetica; COLOR: black;} P,DIV,UL,OL,BLOCKQUOTE{MARGIN-BOTTOM: 0px; MARGIN-TOP: 0px;} BODY{MARGIN: 5px;} &lt;/style>&lt;/HEAD>&lt;BODY> &lt;p style=\'MARGIN-BOTTOM: 0px; margin-top: 0px\' align=\'left\'>&nbsp;&lt;/p>&lt;font style=\'FONT-SIZE: 11pt\'> &lt;p style=\'MARGIN-BOTTOM: 0px; MARGIN-TOP: 0px\'>i. The patient was found to have Anemia and Dr Seven and his staff diagnosed the condition and treated Ms Rebecca for Anemia during the 2 day stay at Community Health Hospitals. Ms Rebecca recovered from Anemia during the stay and is being discharged in a stable condition. If there is fever greater than 101.5 F or onset of chest pain/breathlessness the patient is advised to contact emergency. &lt;/p>&lt;/font>&lt;/BODY>&lt;/HTML>&lt;/li>&lt;/ul>"
```

Comment:
@slvsridhar It may be easier if you post a raw (unescaped) version of the response for @MaxPhilips  to look at. You're right that things need to be escaped for this to be viewed correctly on the web. I think Max is trying to make sure it's still representing what's returned by the API correctly when viewed in the web page (this can get tricky)